projectEuler/problem5.js

```diff
@@ -0,0 +1,28 @@
+// prompt: find the smallest positive number that is evenly divisible by all of the numbers from 1 to 20
+
+// helper function for checking if integer is divisible by all numbers from 1 to 20
+function isDivisible(int) {
+  var count = 0;
+  for (let i = 1; i <= 20; i++) {
+    if (int % i === 0) {
+      count++;
+    }
+  }
```

Comment:
In this loop we check if the int is divisible by all 20 digits even if the first digit fails. Also, we know for sure that a given number is divisible by 1, so let's skip that. We can `break` out of the loop early if a divisibility check fails. We also know that the number must be even to be divisible by 20 so it will certainly be divisible by 2. I'm thinking:

````
for (let i = 3; i <= 20; i++) {
    if (int % i !== 0) break;
    count++;
}
````

Reply:
I tried to optimize isDivisible() with your suggestions; I thought about the isPrime() helper function we worked on together and decided to just have the function return false if ever `int % i !==0`, and return true otherwise. I think this would work, but let me know if I'm missing something important here.

 ```
function isDivisible(int) {
  for (let i = 3; i <= 20; i++) {
    if (int % i !== 0) {
      return false;
    }
  }
  return true;
}
```